src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js

```diff
@@ -496,6 +496,16 @@
         });
       }, 'CustomAttributeDefinition')
     },
+    // used by the Custom Roles admin panel tab
+    Roleable: {
+      custom_role_definitions: CustomFilter(
+        'WHAT_HERE???',  // TODO: or a different mapper helper? which?
```

Comment:
You can also pupulate your data by doing 

```javascript
var acr = new CMS.Models.AccessControlRole({
  name: 'test role',
  read: true
});
acr.save();
```

This is also a good way to verify that your frontend model is set up correctly.